Internal Server Error: /
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/templatetags/static.py", line 116, in render
    url = self.url(context)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/templatetags/static.py", line 113, in url
    return self.handle_simple(path)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/templatetags/static.py", line 129, in handle_simple
    return staticfiles_storage.url(path)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/contrib/staticfiles/storage.py", line 203, in url
    return self._url(self.stored_name, name, force)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/contrib/staticfiles/storage.py", line 182, in _url
    hashed_name = hashed_name_func(*args)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/contrib/staticfiles/storage.py", line 513, in stored_name
    raise ValueError(
ValueError: Missing staticfiles manifest entry for 'styles/styles.css'
Internal Server Error: /api/log-content
Internal Server Error: /api/log-content
Internal Server Error: /api/log-content
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Internal Server Error: /api/log-content
Internal Server Error: /api/log-content
Internal Server Error: /api/log-content
Internal Server Error: /
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/templatetags/static.py", line 116, in render
    url = self.url(context)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/templatetags/static.py", line 113, in url
    return self.handle_simple(path)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/templatetags/static.py", line 129, in handle_simple
    return staticfiles_storage.url(path)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/contrib/staticfiles/storage.py", line 203, in url
    return self._url(self.stored_name, name, force)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/contrib/staticfiles/storage.py", line 182, in _url
    hashed_name = hashed_name_func(*args)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/contrib/staticfiles/storage.py", line 513, in stored_name
    raise ValueError(
ValueError: Missing staticfiles manifest entry for 'styles/styles.css'
Internal Server Error: /
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/templatetags/static.py", line 116, in render
    url = self.url(context)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/templatetags/static.py", line 113, in url
    return self.handle_simple(path)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/templatetags/static.py", line 129, in handle_simple
    return staticfiles_storage.url(path)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/contrib/staticfiles/storage.py", line 203, in url
    return self._url(self.stored_name, name, force)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/contrib/staticfiles/storage.py", line 182, in _url
    hashed_name = hashed_name_func(*args)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/contrib/staticfiles/storage.py", line 513, in stored_name
    raise ValueError(
ValueError: Missing staticfiles manifest entry for 'styles/styles.css'
Internal Server Error: /
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/templatetags/static.py", line 116, in render
    url = self.url(context)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/templatetags/static.py", line 113, in url
    return self.handle_simple(path)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/templatetags/static.py", line 129, in handle_simple
    return staticfiles_storage.url(path)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/contrib/staticfiles/storage.py", line 203, in url
    return self._url(self.stored_name, name, force)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/contrib/staticfiles/storage.py", line 182, in _url
    hashed_name = hashed_name_func(*args)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/contrib/staticfiles/storage.py", line 513, in stored_name
    raise ValueError(
ValueError: Missing staticfiles manifest entry for 'styles/styles.css'
Internal Server Error: /api/log-content
Internal Server Error: /api/log-content
Internal Server Error: /
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/templatetags/static.py", line 116, in render
    url = self.url(context)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/templatetags/static.py", line 113, in url
    return self.handle_simple(path)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/templatetags/static.py", line 129, in handle_simple
    return staticfiles_storage.url(path)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/contrib/staticfiles/storage.py", line 203, in url
    return self._url(self.stored_name, name, force)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/contrib/staticfiles/storage.py", line 182, in _url
    hashed_name = hashed_name_func(*args)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/contrib/staticfiles/storage.py", line 513, in stored_name
    raise ValueError(
ValueError: Missing staticfiles manifest entry for 'styles/styles.css'
Internal Server Error: /api/log-content
Internal Server Error: /api/log-content
Internal Server Error: /api/log-content
Internal Server Error: /api/log-content
Internal Server Error: /api/log-content
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: 'www.onlyfindwebhook193726.site'. You may need to add 'www.onlyfindwebhook193726.site' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'www.onlyfindwebhook193726.site'. You may need to add 'www.onlyfindwebhook193726.site' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/ec2-user/MuscleFuel/venv/lib64/python3.9/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '13.60.248.197'. You may need to add '13.60.248.197' to ALLOWED_HOSTS.
